lib/pkgr/config.rb

```diff
@@ -116,6 +116,10 @@ def env
       @table[:env].is_a?(Pkgr::Env) ? @table[:env] : Pkgr::Env.new(@table[:env])
     end
 
+    def buildpacks
+      @table[:buildpack].is_a?(String) ? @table[:buildpack].split(",") : @table[:buildpack]
```

Comment:
@chytreg the possibility to specify env variables along a buildpack (in the `data/` folder was not a good idea, and you're better served using `--env` for that, so I'm fine with comma-separated list.